### PR TITLE
CIT-41 #time 6h Learn and implement the token based authentication an…

### PR DESCRIPTION
**Pull request by codal-uchoudhary**
…d authorize permissions such as AllowAny and IsAuthenticated

### CODE REVIEW

**Comment by codal-uchoudhary** [comment]
Token based authentication is implemented in the Blog project

**Comment by arvish-codal** [comment]
@codal-uchoudhary , from now on pls mention the topics covered point by point in the PR description.

You have already done it once here --> https://github.com/codal-uchoudhary/python-training-2024/pull/14

So, pls **follow that in each and every PR** from now on.

File: Projects/My_website/myWebsite/myWebsite/settings.py [hunk 1]
```diff
@@ -134,3 +135,12 @@
 # https://docs.djangoproject.com/en/5.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+REST_FRAMEWORK = {
+    'DEFAULT_AUTHENTICATION_CLASSES': [
+        'rest_framework.authentication.BasicAuthentication',
+    ],
```
Comment (arvish-codal) [inline Projects/My_website/myWebsite/myWebsite/settings.py]: @codal-uchoudhary , **as discussed in person as well as in our 1:1**, you need to implement TokenAuthentication and not the Basic Authentication. 

So, pls update accordingly at each and every place.

**Comment by codal-uchoudhary** [comment]
yes , this will become good approach. i will keep in mind while creating new PR from now

File: Projects/My_website/myWebsite/myWebsite/settings.py (hunk 1 above)
Comment (codal-uchoudhary) [reply Projects/My_website/myWebsite/myWebsite/settings.py]: Token based Authentication is Implemented in the Blog project

**Review by arvish-codal — APPROVED** [review]

**Pull request merged**